cmd/instance_show.go

```diff
@@ -93,106 +106,139 @@ func (c *instanceShowCmd) cmdRun(cmd *cobra.Command, _ []string) error {
 
 		return nil
 	}
+	var ipV6 *net.IP
+	if parsed := net.ParseIP(instance.Ipv6Address); parsed != nil {
+		ipV6 = &parsed // only assign pointer if it's a valid IP
+	}
```

Comment:
```suggestion
	ipV6 := net.ParseIP(instance.Ipv6Address)
		
```

https://pkg.go.dev/net#ParseIP
ParseIP will return nil if the IP is not correct so no need to prepare a pointer